simpleline/render/io_manager.py

```diff
@@ -246,28 +248,38 @@ def _thread_input(self, prompt, hidden):
         :param hidden: whether typed characters should be echoed or not
         :type hidden: bool
         """
-        if hidden:
-            data = self._getpass_func(prompt)
+        if isinstance(prompt, Prompt):
+            text_prompt = self._prompt_to_text(prompt)
         else:
-            widget = TextWidget(str(prompt))
-            widget.render(self._width)
-            lines = widget.get_lines()
-            sys.stdout.write("\n".join(lines) + " ")
+            text_prompt = prompt
```

Comment:
Why do we render only the instances of `Prompt` now?

Reply:
We are able to render `Prompt` and `str`. I don't think there is any other option we want to support.

Reply:
It could be removed and just changed to string by `str()`. 
Fixed.